docs/source/index.mdx

```diff
@@ -17,19 +17,19 @@ Let Datasets Server take care of the heavy lifting so you can use a simple **RES
   <img
     style="margin-bottom: 0;"
     class="block dark:hidden"
-    src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/datasets-server/oasst1_light.png"
+    src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/datasets-server/openbookqa_light.png"
```

Comment:
Is it me, or do the two new screenshots have a low-quality resolution?

Reply:
idk this is how my laptop does screenshots...

Reply:
I'll try to make higher resolution screenshots and upload them to huggingface/documentation-images

Reply:
done: 

https://huggingface.co/datasets/huggingface/documentation-images/blob/main/datasets-server/openbookqa_dark.png 

https://huggingface.co/datasets/huggingface/documentation-images/blob/main/datasets-server/openbookqa_light.png

Reply:
I also updated https://huggingface.co/docs/hub/datasets-viewer

From

<img width="1435" alt="Capture d’écran 2023-10-09 à 15 26 06" src="https://github.com/huggingface/datasets-server/assets/1676121/6a1b46df-ab1a-4a0b-842c-44d67a07d31b">

To

<img width="1434" alt="Capture d’écran 2023-10-09 à 15 26 14" src="https://github.com/huggingface/datasets-server/assets/1676121/d1dce651-2e8d-4fbe-8f9e-bb40a5c37355">

---


https://huggingface.co/datasets/huggingface/documentation-images/commit/a5db8a891e22405e0d35cc549868281a1a0d75ca

Reply:
awesome thank you!